types/block_meta.go

```diff
@@ -1,16 +1,18 @@
 package types
 
-// BlockMeta contains meta information about a block - namely, it's ID and Header.
+// BlockMeta contains meta information.
 type BlockMeta struct {
-	BlockID BlockID `json:"block_id"` // the block hash and partsethash
-	Header  Header  `json:"header"`   // The block's Header
+	BlockID BlockID `json:"block_id"`
+	Header  Header  `json:"header"`
+	NumTxs  int64   `json:"number_txs"`
```

Comment:
For consistency, this should probably be `num_txs`. Eg. grep for `num_` vs. `number_` ...